### ISSUE TITLE
Did not received the email with the discount code.

### ISSUE THREAD

**Issue by apeksha10** [TycheSoftwares/order-delivery-date-for-woocommerce#299]
**Describe the bug**
Did not received the email with the discount code.

**To Reproduce**
1. On my local I have upload 1st-time Ordd lite plugin
2. Activate the plugin
3. Click on the Allow button but the discount email did not receive which is an issue.

**Expected behavior**
The discount email should receive .

**Comment by ashokrane** [comment]
@apeksha10 Can you confirm once if this is still an issue?

**Comment by apeksha10** [comment]
Yes..I have checked and confirm that with the live copy, the above issue still present.
Followed the same steps as an issue has.

**Comment by apeksha10** [comment]
With the latest git copy, the above issue is still replicated. 
Discount coupon code email does not receive.
